src/expr/src/scalar/func.rs

```diff
@@ -2071,6 +2071,18 @@ where
     }
 }
 
+fn date_trunc_interval<'a>(a: Datum, mut int: Interval) -> Result<Datum<'a>, EvalError> {
```

Comment:
The convention is that this `int` should be `b: Datum` and then unwrap internally. Also, it's not idiomatic to refer to an interval as an `int` which sound vaguely like you'd expect an interval value; we usually just spell out `interval`.